application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksSystem.java

```diff
@@ -0,0 +1,185 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.channel.ChannelType;
+import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
+import net.dv8tion.jda.api.entities.channel.unions.MessageChannelUnion;
+import net.dv8tion.jda.api.events.interaction.command.GenericCommandInteractionEvent;
+
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.records.BookmarksRecord;
+
+import javax.annotation.Nullable;
+
+import java.awt.*;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+
+import static org.togetherjava.tjbot.db.generated.tables.Bookmarks.BOOKMARKS;
+
+/**
+ * The bookmarks system provides methods to interact with the database. It also enables the
+ * {@link BookmarksCommand} to request paginations from the {@link BookmarksPaginatorInteractor}.
+ */
+public final class BookmarksSystem {
+
+    public static final Color COLOR_SUCCESS = new Color(166, 218, 149);
+    public static final Color COLOR_WARNING = new Color(245, 169, 127);
+    public static final Color COLOR_FAILURE = new Color(238, 153, 160);
```

Comment:
this is a rather unconvential way of writing colors. its hard for people to see what those colors are. id suggest to change it to hex

Reply:
I do not agree. If you create a color with new rather than decode intellij previews the color. Also if you are not using a very simple color you still can't really imagine what it looks like, at least me

Reply:
> this is a rather unconvential way of writing colors. its hard for people to see what those colors are. id suggest to change it to hex

Completely disagree, I know shit about hex, if someone gives me a RGB colour I can make a guess and maybe even partially know the colour it is. Will I ever do that irl? No because IntelliJ shows the colour.

I know multiple RGB colours out of my head (cause in kindergarten they teach you the rainbow), but HEX you only know if you get taught HEX.

I'm fine with it being changed to HEX, but the "its hard for people to see what those colors are" I do not see at all.

I'd personally keep it RGB too, IntelliJ shows a preview either way

Reply:
> I'd personally keep it RGB too, IntelliJ shows a preview either way

@Tais993 For me .decode did not give any preview

Reply:
its important that intellij gives a preview. unfortunately i didnt see a preview for most of the colors in the current code base. 

in either case, it would be cool if we can streamline the color usage across the code. but the better place for such a change would for sure be the color-pallete PR.

so welp, i guess its fine for this PR.

Reply:
I always keep my branch up to date with develop when pushing new changes. This means i will swap to the Color palette if the palette PR gets merged before this feature which is very likely